app/src/main/java/de/westnordost/streetcomplete/quests/building_type/BuildingType.kt

```diff
@@ -58,6 +58,7 @@ import de.westnordost.streetcomplete.quests.building_type.BuildingType.TRANSPORT
 import de.westnordost.streetcomplete.quests.building_type.BuildingType.UNIVERSITY
 import de.westnordost.streetcomplete.quests.building_type.BuildingType.WAREHOUSE
 
+@Suppress("ktlint:paren-spacing")
```

Comment:
I am against validator-specific comments, I would prefer entire rule to be disabled

Reply:
Okay, I'll disable the rule globally and remove the `@Suppress` comments.